src/Microsoft.AspNetCore.Server.Kestrel.Core/Features/IHttpMinRequestBodyDataRateFeature.cs

```diff
@@ -6,7 +6,7 @@ namespace Microsoft.AspNetCore.Server.Kestrel.Core.Features
     /// <summary>
     /// Represents a minimum data rate for the request body of an HTTP request.
     /// </summary>
-    public interface IHttpRequestBodyMinimumDataRateFeature
+    public interface IHttpMinRequestBodyDataRateFeature
```

Comment:
I already change the property/param to bytesPerSecond, so that's taken care of. I'll change the type name now.

I went with Min because we already have Max on other properties.